packages/grafana-data/src/transformations/transformers/order.ts

```diff
@@ -7,8 +7,28 @@ import { DataTransformerInfo } from '../../types/transformations';
 
 import { DataTransformerID } from './ids';
 
+export enum FieldOrdering {
+  Manual = 'manual',
+  Auto = 'auto',
+}
+
+export enum Order {
+  Off = 'off',
+  Asc = 'asc',
+  Desc = 'desc',
+}
+
+// this is a label sort if labelName is defined, and the field name sort if undefined
```

Comment:
This felt a little hacky to me, so I had originally had it where field name sort was separate from label sorts, but I got more annoyed with splitting/rejoining the options than using a string field being undefined or not as a distinction.